core/java/src/org/openda/exchange/dataobjects/GenericNetcdfConfig.java

```diff
@@ -0,0 +1,13 @@
+package org.openda.exchange.dataobjects;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class GenericNetcdfConfig {
+
+//	ArrayList<GenericNetcdfTSConfig> tsConfigs = new ArrayList<>();
+//	ArrayList<GenericNetcdfArrayConfig> arrayConfigs = new ArrayList<>();
+
+	HashMap<String, GenericNetcdfTSConfig> tsConfigs = new HashMap<>();
+	HashMap<String, GenericNetcdfArrayConfig> arrayConfigs = new HashMap<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=4902599)